bin/run.sh

```diff
@@ -89,66 +124,70 @@ else
     echo "Created 'scitran' Virtualenv at $SCITRAN_RUNTIME_PATH"
 fi
 
-if [ -f "$SCITRAN_RUNTIME_PATH/bin/mongod" ]; then
-    echo "MongoDB is installed"
-else
-    echo "Installing MongoDB"
-    curl https://fastdl.mongodb.org/osx/mongodb-osx-x86_64-3.0.7.tgz | tar xz -C $SCITRAN_RUNTIME_PATH --strip-components 1
-    echo "MongoDB installed"
-fi
-
-
 echo "Activating Virtualenv"
 source $SCITRAN_RUNTIME_PATH/bin/activate
 
-echo "Installing Python requirements"
-bin/install.sh
+# Set python path so scripts can work
+export PYTHONPATH=.
 
 
-# Launch mongod
-mongod --dbpath $SCITRAN_PERSISTENT_DB_PATH --smallfiles --port $SCITRAN_PERSISTENT_DB_PORT &
-MONGO_PID=$!
 
-# Set python path so scripts can work
-export PYTHONPATH=.
+###### STARTUP TASKS
+if [[ $START == "1" || $DEV == "1" ]]; then
```

Comment:
They can - I just thought this was easier to read.  The boolean OR has to be a "-o" as well.